airflow/config_templates/default_airflow.cfg

```diff
@@ -592,6 +592,9 @@ hide_sensitive_variable_fields = True
 elasticsearch_host =
 elasticsearch_log_id_template = {{dag_id}}-{{task_id}}-{{execution_date}}-{{try_number}}
 elasticsearch_end_of_log_mark = end_of_log
+elasticsearch_write_stdout = False
+elasticsearch_json_format = False
+elasticsearch_json_fields = asctime, filename, lineno, levelname, message
```

Comment:
We might need some comments around this, e.g. where these fields are from and how they're gonna be used.

Reply:
@KevinYang21 good call! Added some comments, hope they help!